src/cmd-generate-release-meta

```diff
@@ -137,15 +137,20 @@ def append_build(out, input_):
                     }
                 }
             }
-
-    # AMI specific additions
-    if input_.get("amis", None) is not None:
-        arch_dict["media"]["aws"] = arch_dict["media"].get("aws", {})
-        arch_dict["media"]["aws"]["images"] = arch_dict["media"]["aws"].get("images", {})
-        for ami_dict in input_.get("amis"):
-            arch_dict["media"]["aws"]["images"][ami_dict["name"]] = {
-                "image": ami_dict["hvm"]
-            }
+    # Aliyun/AWS specific additions
+    for cloud in ["aliyun", "amis"]:
```

Comment:
`input_` uses `amis` and `arch_dict` uses `aws`.  I'd consider doing

```
for meta_key, cloud, image_field in ("aliyun", "aliyun", "id"), ("amis", "aws", "hvm"):
```

Reply:
Ah good catch; too bad we aren't  more consistent.